queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-30';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts > 3;
```

Comment:
- [ ] It would be better if we improve our query to match the requirements, here we want `escape_attempts < 3` and not `escape_attempts > 3` as we have currently
![Screenshot from 2023-05-08 15-41-52](https://user-images.githubusercontent.com/46388113/236866761-95ef4bc4-a497-4b2b-9acb-ef8d5475c7af.png)

Reply:
thanks @addod19  you are right, I just fixed it